addons/binding/org.openhab.binding.meterreader/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openhab.binding</groupId>
+		<artifactId>pom</artifactId>
+		<version>2.3.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.openhab.binding.meterreader</artifactId>
+
+	<name>MeterReader Binding</name>
+	<packaging>eclipse-plugin</packaging>
+
+	<build>
```

Comment:
please remove the whole `<build>` section - it should not be required as we are using Tycho.

Reply:
I used the maven-dependency-plugin to get 3rd party libs which are no OSGi bundles (and not been used elsewhere) into the bundle via Bundle-ClassPath.
This is the way I'm always doing in my tycho projects. 
How else should I get the jsml.jar?

Reply:
Actually I saw that jsml.jar is an OSGi bundle. 
Would you like to integrate it into openhab distribution?

Reply:
Please see https://www.eclipse.org/smarthome/documentation/development/bindings/dependencies.html#3rd-party-libraries.
We include the binary jar in the repo (I know, that's far from ideal, but that's how it is right now for all bindings) as otherwise people will see compilation problems after IDE setup and would require a Maven build first in order to solve those.
So manually adding the jar to your bundle (even if it is an OSGi bundle) would be the simplest way for you right now.

Reply:
ok, got it!

Reply:
This is left because of the workaround with the jrxtx library. I could omit the `maven-dependency-plugin` section and include the necessary classes in the source folder. But I don't want to include them in the standard source folder but rather in a "java-extern" folder in order to state that they are not belonging directly to the binding.
In order to achieve that, I use the `maven-build-helper` plugin to include them.

Reply:
Not sure I understand - what's the problem with including it as a jar as mentioned in https://github.com/openhab/openhab2-addons/pull/2167#discussion_r178799250? Shading a class should still be possible by choosing the right order for the classpath entry in the bundle manifest.

Reply:
I've copied now all the classes from jrxtx into `src/main/java-extern` (as nearly all of them were dependent on `gnu.io`) and modified them appropriately.  Further I used the `build-helper-maven-plugin` to add that folder to the sources to compile.